lib/node_modules/@stdlib/math/base/special/hyp2f1/lib/utils.js

```diff
@@ -0,0 +1,72 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var round = require( '@stdlib/math/base/special/round' );
+var abs = require( '@stdlib/math/base/special/abs' );
+
+
+// VARIABLES //
+
+var EPS = 1.0e-13;
+
+
+// FUNCTIONS //
+
+/**
+* Tests if a finite double-precision floating-point number is a "close enough" nonpositive integer.
+*
+* @private
+* @param {number} x - input value
+* @returns {boolean} boolean indicating whether the value is a "close enough" nonpositive integer
+*/
+function isNonPositiveInteger( x ) {
+	var diff;
+	var ix;
+
+	ix = round( x );
+	diff = abs( x - ix );
+	return ( ( ix <= 0.0 ) && ( diff < EPS ) );
+}
+
+/**
+* Tests if a finite double-precision floating-point number is a "close enough" integer.
+*
+* @private
+* @param {number} x - input value
+* @returns {boolean} boolean indicating whether the value is a "close enough" integer
+*/
+function isInteger( x ) {
+	var diff;
+	var ix;
+
+	ix = round( x );
+	diff = abs( x - ix );
+	return ( diff < EPS );
+}
+
+
+// EXPORTS //
+
+module.exports = {
+	'isNonPositiveInteger': isNonPositiveInteger,
```

Comment:
@anandkaranubc In general, we prefer having one function/class per file. In which case, these should be modularized accordingly. If you were concerned about duplicating `EPS`, well, that can also go in a separate file and then be imported by the two modules. Yes, this is more files, but follows project conventions.

Reply:
It is also fine to simply duplicate `EPS` in both modules.